src/index.html

```diff
@@ -6,8 +6,41 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <a href="#" class="card__link">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac"
+          class="card__img"
+        >
+      </a>
+
+      <h2 class="card__text card__text--main">APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__text card__text--code">Product code: 195434</p>
+
+
+
+      <div class="card__rating">
+        <div class="card__rating card__rating--star"></div>
+        <div class="card__rating card__rating--star"></div>
+        <div class="card__rating card__rating--star"></div>
+        <div class="card__rating card__rating--star"></div>
+        <div class="card__rating card__rating--star"></div>
+
```

Comment:
you can make it as separate block 
```suggestion

      <div class="card__rating">
       <div class="stars stars--4">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```
just reuse all code from stars task :)